tests/puppeteer/campaigns/functional/BO/04_customers/02_addresses/03_addressesBulkActions.js

```diff
@@ -0,0 +1,104 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const AddressesPage = require('@pages/BO/customers/addresses');
+const AddAddressPage = require('@pages/BO/customers/addresses/add');
+// Importing data
+const AddressFaker = require('@data/faker/address');
+
+let browser;
+let page;
+let numberOfAddresses = 0;
+const addressData = new AddressFaker({address: 'todelete', email: 'pub@prestashop.com', country: 'France'});
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    addressesPage: new AddressesPage(page),
+    addAddressPage: new AddAddressPage(page),
+  };
+};
+
+// Create addresses then delete with Bulk actions
+describe('Create Addresses then delete with Bulk actions', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+  // Login into BO and go to addresses page
+  loginCommon.loginBO();
+
+  it('should go to \'Customers>Addresses\' page', async function () {
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.customersParentLink,
+      this.pageObjects.boBasePage.addressesLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.addressesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.addressesPage.pageTitle);
+  });
+
+  it('should reset all filters', async function () {
+    numberOfAddresses = await this.pageObjects.addressesPage.resetAndGetNumberOfLines();
+    await expect(numberOfAddresses).to.be.above(0);
+  });
+  // 1 : Create 2 addresses in BO
+  describe('Create 2 addresses in BO', async () => {
+    const tests = [
+      {args: {addressToCreate: addressData}},
+      {args: {addressToCreate: addressData}},
```

Comment:
You create twice the same address ? That's not an issue ?

Reply:
Nope, it creates two addresses to properly test the bulk action.

Reply:
Yes, I understood that correctly. I'm just saying there's no issue with creating twice the same address with the same name ?

Reply:
No, you can have the same address multiple time :)
It's logic because you can be a family, with multiple account, but living in the same house.